_posts/2020-06-25-changelog-125.markdown

```diff
@@ -0,0 +1,176 @@
+
+<!-- https://github.com/intellij-rust/intellij-rust/pull/5547 -->
+* Don't trigger `Main function not found` inspection if the file has top-level errors (by [@Kobzol])
+
+<!-- https://github.com/intellij-rust/intellij-rust/pull/5538 -->
+* Fix `Go to Declaration` from associated type bindings on 2020.2
+
+<!-- https://github.com/intellij-rust/intellij-rust/pull/5574 -->
+* Fix parentheses completion for Fn* traits (by [@Kobzol])
+
+<!-- https://github.com/intellij-rust/intellij-rust/pull/5528 -->
+* Liveness analysis takes into account destructured struct fields (by [@Kobzol])
+
+<!-- https://github.com/intellij-rust/intellij-rust/pull/5523 -->
+* Fix `Join Lines` action for doc comment lines (by [@Stzx])
+
+<!-- https://github.com/intellij-rust/intellij-rust/pull/5515 -->
+* Disable `Promote Module to Directory` and `Downgrade Module to File` refactorings for invalid files in the editor (by [@Kobzol])
+
+<!-- https://github.com/intellij-rust/intellij-rust/pull/4868 -->
+* Correctly handle symlinked Cargo projects (by [@Jezza])
+
+<!-- https://github.com/intellij-rust/intellij-rust/pull/5603 -->
+* Take into account namespaces when name shadowing occurs
+
+<!-- https://github.com/intellij-rust/intellij-rust/pull/5623 -->
+* Fix `#[macro_export(local_inner_macros)]` when used with the experimental macro expansion engine
+
+<!-- https://github.com/intellij-rust/intellij-rust/pull/5597 -->
+* Fix macro expansion infinite progress on 2020.2
+
+<!-- https://github.com/intellij-rust/intellij-rust/pull/5640 -->
+* Properly render types in annotations on 2020.2
+
+<!-- https://github.com/intellij-rust/intellij-rust/pull/5611 -->
+* Properly escape text in a test output pane
+
+<!-- https://github.com/intellij-rust/intellij-rust/pull/5549 -->
+* Get rid of options to disable the Test Tool window
+
+* Fix some exceptions:
+  <!-- https://github.com/intellij-rust/intellij-rust/pull/5584 -->
+  * In `Remove reference` and `Change return type` quick fixes
+  <!-- https://github.com/intellij-rust/intellij-rust/pull/5591 -->
+  * When analysing file with an external linter
+  <!-- https://github.com/intellij-rust/intellij-rust/pull/5583 -->
+  * During code folding of invalid macro calls
+  <!-- https://github.com/intellij-rust/intellij-rust/pull/5578 -->
+  * On `Find Usages` in Macros 2.0
+  <!-- https://github.com/intellij-rust/intellij-rust/pull/5595 -->
+  <!-- https://github.com/intellij-rust/intellij-rust/pull/5590 -->
+  * In structure view and usage tooltip
+  <!-- https://github.com/intellij-rust/intellij-rust/pull/5596 -->
+  * When formatting file with rustfmt on save
+
+## Internal Improvements
+
+<!-- https://github.com/intellij-rust/intellij-rust/pull/5545 -->
+<!-- https://github.com/intellij-rust/intellij-rust/pull/5506 -->
+* Now Rust plugin is [dynamically loadable](https://www.jetbrains.org/intellij/sdk/docs/basics/plugin_structure/dynamic_plugins.html) on 2020.2.
+It allows you to install and update the plugin without restarting the IDE
```

Comment:
Let's avoid `update` here because update should unload plugin first and it may not work